subprojects/ide/src/crossVersionTest/groovy/org/gradle/plugins/ide/tooling/r211/ToolingApiEclipseModelCrossVersionSpec.groovy

```diff
@@ -21,6 +21,7 @@ import org.gradle.integtests.tooling.fixture.TargetGradleVersion
 import org.gradle.integtests.tooling.fixture.ToolingApiSpecification
 import org.gradle.tooling.model.UnsupportedMethodException
 import org.gradle.tooling.model.eclipse.EclipseProject
+import static org.gradle.plugins.ide.tooling.r210.ToolingApiEclipseModelCrossVersionSpec.javaTargetCompatibility
```

Comment:
Maybe a "version neutral" location for this utility method would be better? Feels weird to have such an import from "...r210..." in many different tests.